lib/spack/spack/cmd/style.py

```diff
@@ -306,15 +333,21 @@ def prefix_relative(path):
         if not file_list:
             file_list = changed_files(args.base, args.untracked, args.all)
         print_style_header(file_list, args)
+        if args.isort:
+            # We run isort before flake8, assuming that #23947 with flake8-import-order
+            # is also in, to allow flake8 to double-check the result of executing isort,
+            # if --fix was provided.
```

Comment:
We should run both `black` and `isort` before `flake8` since they may fix things that `flake8` will complain about.

Reply:
consolidaetd